app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is defined as a class variable. This means it will be shared across all instances of the `Person` class. If this is not the intended behavior, consider using an instance variable instead.